controllers/about/index.js

```diff
@@ -13,5 +21,131 @@ module.exports = (pages, sectionPath, sectionId) => {
         sectionPath: sectionPath,
         sectionId: sectionId
     });
+
+    const ebulletinPath = sectionPath + pages.ebulletin.path;
```

Comment:
I'd pull this out into a separate file, like we've done in `/funding/index.js`, e.g.

```js
ebulletinRoute.init({
    router: router,
    routeConfig: pages.ebulletin
});
```

Reply:
Good call – take a look now.